web-service-consumer/1.6/modules/ROOT/pages/web-service-consumer-examples.adoc

```diff
@@ -92,8 +92,8 @@ image::web-service-consumer-example6.png[Web Service Consumer Configuration]
 . Save your changes in Studio.
 . Deploy the application.
 
-== Web Service Consumer Example in XML
-In the *Configuration XML* tab in Studio, the XML looks like this (if needed, change the values to reflect your environment):
+== XML for Consuming Web Service
```

Comment:
The header is not clear (but maybe that's because I am not viewing the context). Should it be "XML for the Web Service Consumer Example"?